### ISSUE TITLE
Changing the IP of master does not seem to be full scope

### ISSUE THREAD

**Issue by jmlambert78** [openshift-evangelists/vagrant-origin#50]
I have deployed this vm in a windows virtualbox and get issues on teh IP 10.2.2.2
I found that we may parameter this in the config.env of vagrant BUT
even if the VM is started, the origin conf files are not setup accordingly...
What is the right procedure  ? to update all that together ?
Thx

**Comment by jfmatth** [comment]
if you search the repo for 10.2.2.2 you'll find where the IP is set in the config.  see if that helps

**Comment by jmlambert78** [comment]
Yes, but I dont see where this is included in the origin master & node yaml files ? so when I reach the new IP, I am rerouted to the 10.2.2.2 for the authentication... for example

**Comment by aduzsardi** [comment]
i'm a newbie user of vagrant and would like to run a testing environment of openshift , cand somebody please tell me how to change the IP address 10.2.2.2 to something that i can access from my PC

thank you

--Edit-
nvm , my virtualbox network didn't have DHCP enabled :)
